app/_locales/en/messages.json

```diff
@@ -1346,6 +1346,9 @@
   "lockTimeTooGreat": {
     "message": "Lock time is too great"
   },
+  "lowGasWarning": {
```

Comment:
I'm wondering if there is a better name we could use here. My worry is that a "low gas warning" will be interpreted as a message that warns the user that their gas fee is too low and that they should set it higher. But that's not quite accurate. It's not a bad thing, it's just informing the user about the implication of setting a low max base or priority fee, i.e., that in doing so, their transaction will receive a low priority relative to other transactions (because that's how the fee market works). So, maybe this should be called `lowPriorityMessage`, `lowPriorityExplanation`, or `lowPriorityWarning`? (also see comment below)

Reply:
make sense I renamed it to `lowPriorityMessage`